tests/tensorflow/README.md

```diff
@@ -0,0 +1,26 @@
+# Tesing NNCF in Tensorflow
+
+### Introduction
+In this folder, there are some files available to test if the nncf module is installed and works properly in your local or server environment. It will test NNCF module with mock datasets(`cifar10` for classification, or `coco2017` for detection & segmentation) and mock model. All datas should be in `tests/tensorflow/data/`. You can test locally using the `pytest` command as shown below.
+
+```
+pytest tests/common tests/tensorflow \
+         --junitxml nncf-tests.xml \
+	 --data tests/tensorflow/data
+```
```

Comment:
Could we run pre-commit tests without `--data` and is it only needed for the others (`test_sanity_sample.py`, ....)?
You would better to separate pre-commit tests with the others.
As far as I know, `test_sanity_sample.py`, .... require additional arguments (e.g. `--data`) to run unlike pre-commit tests.